satpy/readers/geocat.py

```diff
@@ -283,7 +276,12 @@ def get_metadata(self, dataset_id, ds_info):
 
         return info
 
-    def get_dataset(self, dataset_id, ds_info, xslice=slice(None), yslice=slice(None)):
+    def get_dataset(self, dataset_id, ds_info, xslice=None, yslice=None):
+        """Get dataset."""
+        if xslice is None:
+            xslice = slice(None)
+        if yslice is None:
+            yslice = slice(None)
```

Comment:
Oh this is from the older versions of Satpy. We can probably remove the use of these slices entirely.